devlib/target.py

```diff
@@ -832,8 +833,17 @@ def kick_off(self, command, as_root=None):
         else:
             raise ValueError('Background command exited before timeout; got "{}"'.format(output))
 
+    def __setup_list_directory(self):
```

Comment:
I think it may be good to have a short comment here to explain why this is necessary.